server/models/handlers/account.js

```diff
@@ -0,0 +1,27 @@
+/* The goal: Take in the user input from the Account Creation form in 
+the front end UI in order to create a user account in the backend*/
+
+async function createUserAccount(firstName, lastName, type, studentID, email, password, phone) {
+    
+    const user = {
+        type: 'student', //note: 'type' is called 'role' in the UI, but refers to either a worker or a student.
+        studentId: 20764242,
+        firstname: 'Victor',
+        lastname: 'Hugo',
+        email: 'victorhugo@gmail.com',
+        phone: '6476442200'
+
+         /*  Please note: When returning user object, 
+        I have omitted the 'password'. 
+        This is done intentionally for privacy reasons. */
+    };
+
+    return user;
+};
+
+//export the createAccountUser
+module.exports = {
+    createAccountUser
```

Comment:
Hi Nancy, this file does not have the methods `createAccountUser` and `isAccessTokenValid`. Thus, it will generate an error upon starting or when running automated tests. Please remove them and add the `createUserAccount` method to be exported so other files can access it.

Reply:
Hi Amy,

Thank you for clarifying the error: I have deleted the requested lines. I have also ensured that the createUserAccount method is exported properly within this file.

Reply:
Hi Nancy, Yes, that addresses my concerns. Thank you!